perf/auto-qual-test/deploy_updater.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# this will look much better once we have kubectl 1.14 with kustomize support
```

Comment:
If it is an issue we can just create our own image

Reply:
I considered this, but felt it was overly complex, as we would now have an additional image to manage releases of, etc.